darwin/item.py

```diff
@@ -6,6 +6,44 @@
 
 @dataclass(frozen=True, eq=True)
 class DatasetItem:
+    """
+    DatasetItems represent files that can be images or videos which belong to a dataset.
+    Said dataset can be initiated or not.
```

Comment:
Not clear here what "initiated" means

Reply:
I got confused with the second line. Took it away!